src/core/src/OmniTileset.cpp

```diff
@@ -462,12 +476,16 @@ void OmniTileset::reload() {
 
     // Add imagery
     for (const auto& imagery : UsdUtil::getChildCesiumImageryPrims(_tilesetPath)) {
-        addImageryIon(imagery.GetPath());
+        if (imagery.GetPrim().IsA<pxr::CesiumIonImagery>()) {
+            addImageryIon(imagery.GetPath());
+        } else {
+            addImageryPolygon((imagery.GetPath()));
```

Comment:
Good catch.

Issues like this are detected in IDEs for Unreal and Unity. It's somewhat ironic Visual Studio Code doesn't, because it auto-added these parentheses. There's a growing list of these enhancements that if added to Visual Studio Code would improve the dev experience for people coming from Unreal, Unity, and elsewhere.